src/components/App/App.js

```diff
@@ -1,38 +1,41 @@
 import React, { Component } from 'react';
 import queries from '../../utils/queries/queries';
+import { LoginForm } from '../LoginForm/LoginForm';
 
-export class App extends Component {
+class App extends Component {
   state = {
     error: '',
-    users: [],
-    newUserId: 0,
+    users: []
   }
 
   async componentDidMount() {
     try {
       const users = await queries.getUsers();
-      this.setState({ users: users.allUsers })
-    } catch({ message }) {
-      this.setState({ error: message })
+
+      this.setState({ users: users.allUsers });
+    } catch ({ message }) {
+      this.setState({ error: message });
     }
   }
   
-  async createUser() {
+  createUser = async () => {
     const mockUser = {
-      email: "tiff@sum.com",
-      nickname: "ERICCCCCC",
-      password: "asdfasdf",
-    }
-    const newUser = await queries.newUser(mockUser);
+      email: 'tiff@sum.com',
+      nickname: 'ERICCCCCC',
+      password: 'asdfasdf'
+    };
+    
+    await queries.newUser(mockUser);
```

Comment:
We are not currently using newUser anywhere so I removed the unused variable declaration.